corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name = name.capitalize
+    name[0, 10] if name.length > 10
+  end
```

Comment:
- The function will return `nil` if you pass a name where `name.length < 10` and that's not the expected behavior.

- [ ] Kindly fix the issue.

Reply:
This issue has been addressed well. Nice job! :+1: :+1: